_includes/members.html

```diff
@@ -0,0 +1,60 @@
+<section class="slice base">
+    <div class="wp-section">
+        <div class="container">
+            <div class="row">
+                <div class="col-md-12">
+
+                    <div id="carouselTestimonial" class="carousel carousel-testimonials slide" data-ride="carousel">
+                        <ol class="carousel-indicators">
+                            <li data-target="#carouselTestimonial" data-slide-to="0" class="active"></li>
+                            <li data-target="#carouselTestimonial" data-slide-to="1" class=""></li>
+                            <li data-target="#carouselTestimonial" data-slide-to="2" class=""></li>
+                        </ol>
+
+                        <div class="carousel-inner">
+                            <div class="item active">
+                                <div class="text-center">
+                                    <h4><i class="fa fa-quote-left fa-3x"></i></h4>
+                                    <p class="testimonial-text">
```

Comment:
these 3 identical sections should probably get a for loop look at: http://stackoverflow.com/questions/20924471/for-loop-wrap-every-two-posts-in-a-div

and look at collections: https://jekyllrb.com/docs/collections/

a collection can be seen as an array in a loop